index.html

```diff
@@ -35,6 +35,170 @@ <h2 class="green">I’m a software developer! </h2></div>
         <li><a href=""><img src="icons/angellist.svg"></a></li>
       </ul>
     </div>
-      </section>
+    <img src="icons/Disabled.svg" alt="next">
+    <div class="works">
+      <h2>My Recent works</h2>
+      <h2><hr></h2>
+      <div class="container1">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button type="submit">See project</button>
+        </div>
+      </div>
+      <div class="containera">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button type="submit">See project</button>
+        </div>
+      </div>
+      <div class="containerb">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button type="submit">See project</button>
+        </div>
+      </div>
+      <div class="containerc">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button type="submit">See project</button>
+        </div>
+      </div>
+      <div class="containerd">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button type="submit">See project</button>
+        </div>
+      </div>
+      <div class="containere">
+        <div class="img-placeholder">
+        </div>
+        <div class="text">
+          <h2>Multi-Post Stories Gain+Glory</h2>
+          <ul>
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
+          <button type="submit">See project</button>
+        </div>
+      </div>
+      <div class="about">
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly consider replacing this div with section tag to follow best semantics and practices.